backend/src/main/java/gov/cdc/usds/simplereport/utils/TokenAuthentication.java

```diff
@@ -0,0 +1,58 @@
+package gov.cdc.usds.simplereport.utils;
```

Comment:
This doesn't feel like the most idiomatic Spring architecture but I'm not sure where else to put something like this. Suggestions are welcome!

Reply:
Is any of this specific to the actual RS implementation?  If not I think it's fine where it is.